### PR TITLE
feat: Turn EDI premium data model into public data model

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

This PR includes:
- Turn the EDI specific public data model into a general public data model (rename database from `wholesale_edi_results` to `wholesale_calculation_results`)
- Rename column `unit` to `quantity_unit`
- Rename column `qualities` to `quantity_qualities`

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are ~written and~ executed locally
- [x] Subsystem tests have been tested (by manually deploying to `dev_002`) ([deploy here](https://github.com/Energinet-DataHub/dh3-environments/actions/runs/9443682301))

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202406071259_Create__calculation_results__database.sql
Comment (dstenroejl) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202406071259_Create__calculation_results__database.sql:1]: I'm not sure if we need to also delete the old schema/database?

Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202406071259_Create__calculation_results__database.sql:1]: I believe we should. What is your concern?

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405231310_Create_edi_results_database.sql [hunk 1]
```diff
@@ -1 +1,3 @@
-CREATE DATABASE IF NOT EXISTS {EDI_RESULTS_DATABASE_NAME}
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202405231310_Create_edi_results_database.sql:1]: Will this cause problems? Changing script back in time.

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202406071259_Create__calculation_results__database.sql
Comment (dstenroejl) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202406071259_Create__calculation_results__database.sql:1]: I agree we should. The script didn't use the correct name previously though.

**Review by dstenroejl — APPROVED** [review]
Approved, but think it might be a good idea to test it in dev_002

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405231310_Create_edi_results_database.sql (hunk 1 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405231310_Create_edi_results_database.sql:1]: I hope not... The alternative is a bit annoying. Wanna talk about it?

Comment (energinet-ajw) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405231310_Create_edi_results_database.sql:1]: A test on a reset dev_002 would probably show any problems.

Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405231310_Create_edi_results_database.sql:1]: I did it without reset.

**Pull request merged**